Map.cpp

```diff
@@ -0,0 +1,36 @@
+#include "Map.h"
+
+
+Map::Map(std::string filename)
+{
+	std::ifstream file;
+	file.open(filename);
+	std::string line;
+	if (file.is_open()) {
+		while (getline(file, line)) {
+			map.push_back(line);
+		}
+		file.close();
+	}
+	else {
+
+		std::string errMsg = "The file cannot be opened!";
+		throw WrongIndexException(errMsg);
```

Comment:
a WrongIndexException-ben az "index" nem a koordinátákra utal? Nekem fura, hogy itt is ez van meghívva, de ahogy gondoljátok.

Reply:
Jogos, ezt javítottam.